### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by sleshchenko**
### What does this PR do?
This PR contains some extracted changes from https://github.com/che-incubator/chectl/pull/257
Commits are self-explaining, so please review it in commit-by-commit manner.

### What issues does this PR fix or reference?
https://github.com/che-incubator/chectl/pull/257

### CODE REVIEW

File: src/installers/helm.ts [hunk 1]
```diff
@@ -182,6 +182,7 @@ error: E_COMMAND_FAILED`)
   async prepareCheHelmChart(flags: any, cacheDir: string) {
     const srcDir = path.join(flags.templates, '/kubernetes/helm/che/')
     const destDir = path.join(cacheDir, '/templates/kubernetes/helm/che/')
+    await remove(destDir)
```
Comment (l0rd) [inline src/installers/helm.ts]: That makes sense but shouldn't we do the same for every installer (operator and minishift addon)?

Comment (sleshchenko) [reply src/installers/helm.ts]: Let me check. I faced this issue only with helm installer.

Comment (sleshchenko) [reply src/installers/helm.ts]: You're totally right!
I thought that the error happened in `mkdirp` but actually an error happens when there are some conflicting files.
Screenshot_20190823_145921

I'm adding `removeDir` for operator and minishift addon as well.

File: src/flags.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
+/*********************************************************************
```
Comment (l0rd) [inline src/flags.ts]: Maybe this file should be better named: `common-flags.ts`?

File: src/installers/helm.ts (hunk 1 above)
Comment (sleshchenko) [reply src/installers/helm.ts]: Fixed in https://github.com/che-incubator/chectl/pull/301/commits/0b01a4166c490d3a1f6e8be23ea761f80ab1e9ec

File: src/flags.ts (hunk 2 above)
Comment (sleshchenko) [reply src/flags.ts]: Fixed.

**Pull request merged**